tests/manage/pv_services/test_pvc_delete_verify_size_is_returned_to_backendpool.py

```diff
@@ -44,34 +45,39 @@ def check_ceph_used_space():
         return used_in_gb
     used_space = used_in_gb
     raise UnexpectedBehaviour(
-        f"In Ceph status, used size is keeping varying"
+        f"In Ceph status, used size is varying"
     )
 
 
 @retry(UnexpectedBehaviour, tries=5, delay=3, backoff=1)
-def verify_pv_not_exists(pvc_obj, cbp_name):
+def verify_pv_not_exists(pvc_obj, cbp_name, rbd_image_id):
```

Comment:
This function seems redundant. 
Would it make sense to call helpers.verify_volume_deleted_in_backend from within helpers.validate_pv_delete()?

Reply:
@parikshithb Please proceed with the current implementation, we can do this change later as part of a separate PR